websocket/models/models.go

```diff
@@ -360,3 +360,18 @@ type Level2Book struct {
 	// The timestamp that the tick was received by Polygon.
 	ReceivedTimestamp int64 `json:"r,omitempty"`
 }
+
+// IndexValue is value data for either indices.
+type IndexValue struct {
```

Comment:
Is this the name we're using in the API docs too? I don't love adding `Value` at the end of types because it should be self explanatory if a descriptive enough name is chosen

Reply:
`value` is the data type. It is the same class as `trade`, `price`, or `quote`